_includes/nav.html

```diff
@@ -1,57 +1,29 @@
-<nav class="navbar is-transparent">
-    <div class="navbar-brand">
-        <a class="navbar-item" href="/">
-            <img src="../images/rubynepal.png" alt="Ruby Nepal: The Official Ruby Community">
-        </a>
-        <div class="navbar-burger burger" data-target="navbarExampleTransparentExample">
-            <span></span>
-            <span></span>
-            <span></span>
-        </div>
+<nav class="navbar is-transparent rbnw-nav">
+  <div class="navbar-brand">
+    <a class="navbar-item rbnw-nav-logo" href="/">
+      <img src="{{site.url}}/images/rubynepal.png" alt="Ruby Nepal: The Official Ruby Community">
+    </a>
+    <div class="navbar-burger burger is-active" data-target="navbarExampleTransparentExample">
+      <span></span>
+      <span></span>
+      <span></span>
     </div>
+  </div>
 
-    <div id="navbarExampleTransparentExample" class="navbar-menu">
-        <div class="navbar-start">
-            <a class="navbar-item" href="/blogs">
-                Activities
-            </a>
-            <a class="navbar-item" href="/projects">
-                Projects
-            </a>
-            <a class="navbar-item" href="/blogs">
-                Blogs
-            </a>
-            <a class="navbar-item" href="/blogs">
-                Archives
-            </a>
-        </div>
+  <div id="navbarExampleTransparentExample" class="navbar-menu is-active">
+    <div class="navbar-start">
+      <a class="navbar-item" href="/category/activities">Activities</a>
```

Comment:
Underline the links on mouse hover here as well just like in home page

Reply:
Done :)